examples/kafka-hub/hub/modules/config/configurations.bal

```diff
@@ -76,3 +81,11 @@ public final string WEBSUB_EVENTS_CONSUMER_GROUP = os:getEnv("WEBSUB_EVENTS_CONS
 isolated function constructSystemConsumerGroup() returns string {
     return string `websub-events-receiver-${SERVER_IDENTIFIER}-${util:generateRandomString()}`;
 }
+
+isolated function getRetryableStatusCodes(int[] configuredCodes) returns int[]|error {
+    if os:getEnv("RETRYABLE_STATUS_CODES") is "" {
+        return configuredCodes;
+    }
+    string[] statusCodes = re ` `.split(os:getEnv("RETRYABLE_STATUS_CODES"));
```

Comment:
Wouldn't it be better to accept a comma separated list instead? 
Also better to trim the split values before assigning them to the `statusCodes`
```suggestion
    string[] statusCodes = re `,`.split(os:getEnv("RETRYABLE_STATUS_CODES"));
```

Reply:
Fixed with [d4dc22dcceacf106c10e808b38ff86ee940ac075](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/1053/commits/d4dc22dcceacf106c10e808b38ff86ee940ac075)